### PR TITLE
SUPPORTENG-444_syncing_cli_docs

### PR DESCRIPTION
**Pull request by marinahand**

### CODE REVIEW

File: docs/_reference/cli-docs.md [hunk 1]
```diff
@@ -217,6 +217,8 @@ Your Zapier CLI should be installed and ready to go at this point. Next up, we'l
 # create a directory with the minimum required files
 zapier init example-app
 
+> Note: When you run `zapier init`, you'll be presented with a list of templates to start with. Pick the one that matches a feature you'll need (such as "dynamic-dropdown" for an integration with [dynamic dropdown fields](#dynamic-dropdowns)), or select "minimal" for an integration with only the essentials. [View more example apps here](https://github.com/zapier/zapier-platform/tree/main/example-apps).
```
Comment (michellechu77) [inline docs/_reference/cli-docs.md:220]: The preview of the formatting looks wonky there, but I don't necessarily see anything incorrect 🤔 maybe just double check this!

File: docs/_reference/cli-docs.md [hunk 2]
```diff
@@ -1783,7 +1787,7 @@ We provide several methods off of the `z` object, which is provided as the first
 
 ### `z.request([url], options)`
 
-`z.request([url], options)` is a promise based HTTP client with some Zapier-specific goodies. See [Making HTTP Requests](#making-http-requests).
+`z.request([url], options)` is a promise based HTTP client with some Zapier-specific goodies. See [Making HTTP Requests](#making-http-requests). `z.request()` will percent-encode non-ascii characters and these reserved characters: ``:$/?#[]@$&+,;=^@`\``. Use [`skipEncodingChars`](https://github.com/zapier/zapier-platform/blob/main/packages/schema/docs/build/schema.md#requestschema) to modify this behaviour.
```
Comment (michellechu77) [inline docs/_reference/cli-docs.md:1790]: Not critical at all, but might be nice to hyperlink percent-encode with a link to docs like https://developer.mozilla.org/en-US/docs/Glossary/Percent-encoding for those who aren't familiar on what to expect.

**Comment by michellechu77** [comment]
Thanks for making these changes, Marina! Had a couple comments, but nothing critical so feel free to merge as you wish. MR tip for you next time: I'd make it a habit to _always_ leave a PR description. Whether it's just a high-level description of what changes you made, or a quick bulleted list of changes the reviewer should expect to see, any description is super helpful to reviewers to know that they're getting into!

**Review by michellechu77 — DISMISSED** [review]

**Comment by marinahand** [comment]
Thanks @michellechu77! I updated those sections, and appreciate the note on descriptions, will make sure to include 😊

**Review by michellechu77 — APPROVED** [review]

**Pull request merged**